Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.2.0"
 [deps]
 CpuId = "adafc99b-e345-5852-983c-f28acb93d879"
 FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
```

Comment:
Is this needed for access to `AbstractGray`?

Reply:
`ImageCore` reexports `ColorTypes` and `FixedPointNumbers`, it would be convenient to leave compat control to `ImageCore`; use whatever `ColorTypes` and `FixedPointNumbers` versions that `ImageCore` declares compatible.